### ISSUE TITLE
Openh264 createing an issue while upgrading from Fedora38 to Fedora 39

### ISSUE THREAD

**Issue by prathamvbhat** [cisco/openh264#3714]
When i tried to install the latest update that is F39 i got an error message telling that:
```
Cannot download Packages/g/gstreamer1-plugin-openh264-1.22.1-1.fc39.x86_64.rpm: All mirrors were tried; Last error: Interrupted by header callback: Inconsistent server data, reported file Content-Length: 293, repository metadata states file length: 437107 (please report to repository maintainer)
```
Later when i tried using the cli way i got this error:
![image](https://github.com/cisco/openh264/assets/102966477/f578fb58-4eb6-4269-bc41-225c704efec5)

Please let me know thet solutions for it

**Comment by mdd3135** [comment]
i find the same problem too

**Comment by kalev** [comment]
I've seen this before and it seems to be a transient issue with the Cisco server. I am not sure this issue tracker is the best place to report it, but I don't know a better place either :)

It should be possible to work this around with by passing `--disablerepo='*cisco*'` to dnf system-upgrade, which then temporarily disables the repository for the duration of this one command.

**Comment by Prateek-Srivastav** [comment]
is using `--disablerepo='*cisco*'` safe?

**Comment by gopelkujo** [comment]
Exactly same problem with me. Try to upgrade my fedora 37 to 38 and showing this error
![image](https://github.com/cisco/openh264/assets/18652497/78caac47-f814-4643-9ac9-491bee1353aa)

I've been searching for many days for this problem but i don't find one. I will try @kalev temporary solution for upgrading to 38. I'll let you know my result.

**Comment by gopelkujo** [comment]
Hey i just successfully upgrading from fedora 37 to 38 by disabling cisco repo by this line `--disablerepo='*cisco*'`. After all, i still have a problem with installing openH264
![Screenshot from 2023-12-17 12-40-29](https://github.com/cisco/openh264/assets/18652497/215ed25f-40db-467e-8a9d-4ec4258cc0fc)

**Comment by Prateek-Srivastav** [comment]
Maybe you are blocked from accessing ciscobinary website. Can you check if you manually download the package from this url `http://ciscobinary.openh264.org/gstreamer1-plugin-openh264-1.22.1-1.fc38.x86_64.rpm` ?

If you can't, then I'll suggest use a VPN and then download it using the above url and install it manually.

I've updated to 39 using @kalev 's solution and then updated gstreamer manually. I was hitting some sort of firewall or maybe blocked by my ISP or something, but I was blocked, so I used VPN and then downloaded the package.

**Comment by prathamvbhat** [comment]
is this issue related to the internet we use, might be providers have blocked few stuff

**Comment by Prateek-Srivastav** [comment]
Yeah this might be the case that our ISP is blocking some URLs to be accesses, don't know the reason though.

**Comment by gopelkujo** [comment]
Ahh yess, mine fixed by download and install manually using vpn. Thanks!

**Comment by wyup** [comment]
There isn't any openh264 available packages actually on fedora-cisco rawhide repository, I have filed a bug in fedora because it is fedora related, fedora packages openh264, it seems Cisco gives them permission and Cisco already publishes latest 2.4.0. here.
